ext/riverlea/CRM/riverlea/Upgrader.php

```diff
@@ -0,0 +1,137 @@
+<?php
+use CRM_riverlea_ExtensionUtil as E;
+
+/**
+ * Collection of upgrade steps.
+ */
+class CRM_riverlea_Upgrader extends \CRM_Extension_Upgrader_Base {
+
+  // By convention, functions that look like "function upgrade_NNNN()" are
```

Comment:
@ufundo Looks like an upgrader is needed to add the new sql table (and all this boilerplate clutter can be deleted too).

I recently updated the docs about how to do so: https://docs.civicrm.org/dev/en/latest/step-by-step/create-entity/#5b-add-upgrade-step-for-extensions-used-in-production

Reply:
ah thanks that's what I was after!